src/style.css

```diff
@@ -1 +1,51 @@
-/* add styles here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
```

Comment:
The `body` tag is an exception where you can use tag names for styling, so this is acceptable. However, since you have already reset the margin in the `*` selector, this is redundant and can be removed.